### DISCUSSION TITLE
NEWBIE

### DISCUSSION THREAD

**Discussion by RoyGila** [community/community#43119] [New to GitHub] [answered]
Hi, i'm new here. Just trying to explore this world. i'm going to need help as i don't really know what i'm doing here but i'm interested to learn new things.

**Comment by Akash1134** [comment, marked as answer]
👋 Welcome to the GitHub Community on Discussions, @RoyGila ! Happy to have you here 🙂

To get started, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776) and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912).

**Reply by RoyGila** [reply]
Thank you so much!

**Reply by Lexus330v** [reply]
Yoo
